### PR TITLE
Refactored panel fragments and template

### PR DESCRIPTION
**Pull request by matldupont**
- Broke out panel fragments into each respective js file
- moved "data" json files into pages directory and updated gatsby-config
- renamed panels.js template file to panel-page.js
- refactored template to dynamically load from "components" rather than relying on a mapping

### CODE REVIEW

File: src/components/layout.js [hunk 1]
```diff
@@ -1,32 +1,49 @@
...
-        </Box>
-        <Box as="footer">
-          © {new Date().getFullYear()}, Built by
-          {` `}
-          <a href="https://www.rebel.com">Rebel</a>
-        </Box>
-      </Box>
-    </React.Fragment>
-  </ThemeProvider>
+  <StaticQuery
```
Comment (mkallies) [inline src/components/layout.js]: Prefer `useStaticQuery`

File: src/components/panels/ethos.js [hunk 2]
```diff
@@ -36,8 +38,21 @@ const Ethos = ({ data }) => {
   )
 }
 
+Ethos.displayName = 'Ethos'
```
Comment (mkallies) [inline src/components/panels/ethos.js]: I noticed a bunch of these `displayName`s, does React not infer this automatically?

Comment (matldupont) [reply src/components/panels/ethos.js]: Unfortunately, no

File: src/components/layout.js (hunk 1 above)
Comment (matldupont) [reply src/components/layout.js]: Agreed.  Will switch to that

**Pull request merged**